resources/nl-core/nl-core-Procedure.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why is there "Procedure" and "Procedure-ProcedureRequest"? Is the past use case more "true" than the future use case? A name like "zib-Procedure-Procedure" would of course be confusing, but maybe we can do something like zib-Procedure-performed and zib-Procedure-requested?